tests/rkt_auth_test.go

```diff
@@ -43,7 +43,7 @@ const (
 	// result is that /inspect receives two parameters -
 	// "--print-msg='Authentication" and "succeeded.'"
 	authSuccessfulDownload = "AuthenticationSucceeded."
-	authFailedDownload     = "error downloading ACI: bad HTTP status code: 401"
+	authFailedDownload     = "error downloading ACI"
```

Comment:
Does this test pass if we listen only for `bad HTTP status code: 401`? I think I'd prefer to `error downloading ACI`.